websites/N/Notespace/metadata.json

```diff
@@ -4,15 +4,15 @@
 		"id": "630812692659044352",
 		"name": "itsmateo20"
 	},
-	"service": "Notespace",
+	"service": "notespace",
```

Comment:
why

Reply:
its not 
![image](https://github.com/PreMiD/Presences/assets/84156177/600a70ec-2e43-4ecb-acfd-353e6b28b4b0)

Reply:
Note that if you want to correct the capitalization of this, you will also need to rename the folder to `N/notespace` and reset the version to `1.0.0`. This will be considered a deletion and a creation of a new presence.